### PR TITLE
feat(ramp): add buy and sell icon and string

### PR DESCRIPTION
**Pull request by wachunei**
## Explanation

This PR changes the icon and text for the portfolio ramp experience from "Buy" to "Buy & Sell".

## Manual Testing Steps

Clicking on the Buy & Sell button should not change behavior and take the user to the portfolio buy experience.

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [x] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Review by darkwing — DISMISSED** [review]

File: app/_locales/en/messages.json [hunk 1]
```diff
@@ -626,6 +626,9 @@
   "buy": {
```
Comment (DDDDDanica) [inline app/_locales/en/messages.json:626]: Should this be removed if not used anymore?

File: app/_locales/en/messages.json [hunk 2]
```diff
@@ -626,6 +626,9 @@
...
     "message": "Buy"
   },
+  "buyAndSell": {
```
Comment (DDDDDanica) [inline app/_locales/en/messages.json:629]: You might also want to take a look at `ui/components/multichain/select-action-modal/select-action-modal.js` and replace the text there :)

Comment (wachunei) [reply app/_locales/en/messages.json:629]: Great catch! I will need to also request copywriting for the description in that component. Will update soon. Thank you Danica

Comment (wachunei) [reply app/_locales/en/messages.json:629]: Added in [16f95de](https://github.com/MetaMask/metamask-extension/pull/20891/commits/16f95de8894c345b2862d7c8f0ca66ee593fc371)

File: app/_locales/en/messages.json (hunk 1 above)
Comment (wachunei) [reply app/_locales/en/messages.json:626]: I think we could still reference this in the future, but I'll defer to you. I'm ok with removing it.

Comment (DDDDDanica) [reply app/_locales/en/messages.json:626]: Just checked the usage, they should be covered in 3 places, we can remove it :)

Comment (wachunei) [reply app/_locales/en/messages.json]: Oh, it is actually required to be removed https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/57869/workflows/a0945dab-6741-491d-8730-0e16cbc93cf6/jobs/1763269

I ran `yarn run verify-locales:fix` here c0518b8bf1

**Review by darkwing — APPROVED** [review]

**Review by DDDDDanica — APPROVED** [review]

**Pull request merged**